### PR TITLE
Change inserter search placeholder text color

### PR DESCRIPTION
**Pull request by enriquesanchez**
## Description
This PR changes the placeholder text color on the block inserter's search field from #72777c to #6c6f75. The previous color combination [is failing AA success criteria with a ratio of 4.11](https://contrastchecker.com/?c=72777C&b=F3F4F5). New color combination [achieves passing score 4.57](https://contrastchecker.com/?c=6c6f75&b=F3F4F5).

## How has this been tested?
Tested on Firefox/macOS. Contrast ratio tested with https://webaim.org/resources/contrastchecker/ and https://contrastchecker.com

## Screenshots 

Previous (#72777c):

Screen Shot 2020-05-18 at 16 32 35

Proposed (#6c6f75):

Screen Shot 2020-05-18 at 16 27 35

**UPDATE:**

Using `$dark-gray-400` (#606A73) instead. We now have a passing 5.01 contrast ratio.

Screen Shot 2020-05-19 at 11 01 51

## Types of changes
Visual, bug fix.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by enriquesanchez** [comment]
@jasmussen if you don't mind, I would love to have your review this little PR. 

I couldn't find any other place in the UI where this background and foreground color combination was being used, so what I did was find the color combo that just gets us past the success ratio and added it just to his element. 

If this color change proposal is acceptable, I'm sure we'd want to be able to re-use it, so probably creating a variable would be best?

File: packages/block-editor/src/components/inserter/style.scss [hunk 1]
```diff
@@ -88,6 +88,10 @@ $block-inserter-tabs-height: 44px;
 			background: $white;
 			box-shadow: 0 0 0 $border-width-focus $theme-color;
 		}
+
+		&::placeholder {
+			color: #6c6f75;
```
Comment (jasmussen) [inline packages/block-editor/src/components/inserter/style.scss:93]: Can you use `$dark-gray-400` instead? That's #606a73.

**Comment by jasmussen** [comment]
I wish we had a "new gray" for this use case, that is, a color that was created for G2. G2 means to use fewer grays overall, but $medium-gray-text does not meet the contrast ratio here. So for the time being and pending other improvements, seems fine to use a variable from the "old" set. But still important to use a variable, so it's easy to search and replace when we do get a time to revisit.

👍 👍

File: packages/block-editor/src/components/inserter/style.scss (hunk 1 above)
Comment (enriquesanchez) [reply packages/block-editor/src/components/inserter/style.scss]: Thank you @jasmussen! $dark-gray-400 gets us a 5.01 passing score. Much better.

**Comment by enriquesanchez** [comment]
Updated to use `$dark-gray-400`

**Review by jasmussen — APPROVED** [review]
We will want to revisit the grays again, and therefore also revisit this one. 

In the mean time, good to go. Need to run the tests again.

**Comment by enriquesanchez** [comment]
Thank you for the review, @jasmussen!

**Pull request merged**